src/main/java/Main.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Scanner;
+
+public class Main {
+
+
+
+    public static void main(String[] args) {
+
+
+        User u1 = new User("Marek", "marekfrancja@wp.pl");
```

Comment:
remove this. all users should be added via console. Code will be tested using JUnit tests